bin/check-influxdb-query.rb

```diff
@@ -43,13 +43,13 @@
 class CheckInfluxdbQuery < Sensu::Plugin::Check::CLI
   check_name nil
   option :host,
-         short: '-H HOST',
+         short: '-h HOST',
```

Comment:
this conflicts with the -h help flag. --help isn't needed since the CLI library does it automatically so if you removed that -h could be used for host.